packages/ckeditor5-dev-changelog/src/utils/promptreleasetype.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import inquirer from 'inquirer';
+import type { ChangelogReleaseType } from '../types.js';
+
+/**
+ * Prompts the user to choose between latest or prerelease
+ */
+export async function promptReleaseType(): Promise<ChangelogReleaseType> {
+	const { releaseType } = await inquirer.prompt( [
+		{
+			type: 'list',
+			name: 'releaseType',
+			message: 'Select the release type?',
+			choices: [
+				{ name: 'Latest (stable) release', value: 'latest' },
+				{ name: 'Pre-release (alpha/beta/rc)', value: 'prerelease' }
+			]
+		}
+	] );
```

Comment:
Make it a `y/n` question.

> Is it a pre-release? (y/N)

It would be enough to decide what to do.

Reply:
Changed to `y/n` question in [e4f1e73](https://github.com/ckeditor/ckeditor5-dev/pull/1183/commits/e4f1e73009f8cbfb20a274d322f2225d0c5bcd46).